src/components/tasks/card/index.tsx

```diff
@@ -442,15 +509,46 @@ const Card: FC<Props> = ({
                                 height={30}
                             />
                         </span>
-                        <span
-                            className={classNames.cardStrongFont}
-                            contentEditable={shouldEdit}
-                            onKeyPress={(e) => handleChange(e, 'assignee')}
-                            role="button"
-                            tabIndex={0}
-                        >
-                            {cardDetails.assignee}
-                        </span>
+                        {shouldEdit ? (
+                            isUserAuthorized && (
+                                <div
+                                    style={{
+                                        position: 'relative',
+                                        display: 'inline-block',
+                                    }}
```

Comment:
can we use class instead of inline style?

Reply:
This were highly localized style so i went for in-line styles